plugins/se.infomaker.ximconcept/components/ConceptMainComponent.js

```diff
@@ -127,6 +133,31 @@ class ConceptMainComponent extends Component {
         }
     }
 
+    async decorateExistingItemsWithRemoteMeta(existingItems, updatedConcept) {
+        if (Array.isArray(existingItems)) {
+            const enhancedItems = await Promise.all(existingItems.map(async (item) => {
+                const existingItem = this.state ?
+                    this.state.existingItems.find(existingItem => existingItem.uuid === item.uuid) :
```

Comment:
If you want to be cool this could be simplified as `this.state.existingItems.find(({uuid}) => existingItem.uuid === uuid)`
Also, will `this.state` ever not exist?

Reply:
👍